firmware/config/config.h.template

```diff
@@ -59,6 +66,9 @@
 // TRUETYPE FONT CONFIGURATION
 //#define DEFAULT_FONT ROBOTO_REGULAR
 
+// ADVANCED COMPILE OPTIONS
+#define INCLUDE_MDNS // Include mDNS funcionality
```

Comment:
Ok, I'll explain it a bit more (although most people will probably never change it, but I wanted to expose this setting in config.h.template so that it's easier to see that it's configurable)